### PR TITLE
Switch license from CLA to DCO

### PR DESCRIPTION
**Pull request by michawai**
Before submitting a pull request, please ensure the following:

## Description

Please explain the changes you made.

Add a link to the design (if applicable).

### Checklist

- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Linting run locally successfully
- [ ] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,6 +67,10 @@ workflows:
           filters:
             tags:
               only: /^v[0-9]+(\.[0-9]+)*/
+      - compliance:
+          filters:
+            tags:
+              only: /^v[0-9]+(\.[0-9]+)*/
```
Comment (bd82) [inline .circleci/config.yml:73]: why wait for release to run the compliance check?

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: vscode-mta-tools
+Upstream-Contact: Raya Koren <raya.koren@sap.com>
```
Comment (bd82) [inline .reuse/dep5:3]: can we use a DL?

File: .circleci/config.yml (hunk 1 above)
Comment (bd82) [reply .circleci/config.yml:73]: because of the tags section with:
- only: /^v[0-9]+(\.[0-9]+)*

File: .reuse/dep5 (hunk 2 above)
Comment (bd82) [reply .reuse/dep5:3]: ok

**Review by bd82 — APPROVED** [review]

**Review by bd82 — APPROVED** [review]

**Review by rayakoren — APPROVED** [review]

**Pull request merged**